ark/phenotyping/som_utils.py

```diff
@@ -637,6 +733,40 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
     # set seed for subsetting
     np.random.seed(seed)
 
+    # create path for channel normalization values
+    channel_norm_path = os.path.join(base_dir, pre_dir, 'channel_norm.json')
```

Comment:
Just a small comment. We are currently saving the 99.9% normalization values (that existed before) as prefix_norm_vals.feather in the base directory. Should we also change that to a json to be consistent? Or else save these as a feather too? And maybe move that file into the preprocessed directory as well?

Also, I think it would be helpful to change the names of both of these files, so users are not confused which file is corresponding to which normalization. Maybe like "channel_norm" and "post_pixelnorm_channel_norm" or something?